interoptest/src/nodejsservice/src/testservice/service-hopper.js

```diff
@@ -94,6 +116,57 @@ function grpcServiceHop (host, port, request) {
   });
 }
 
+function toBuffer (testRequest) {
```

Comment:
common code. can this be put in a utility file?

Reply:
Done